src/boot/efi/secure-boot.c

```diff
@@ -35,3 +37,87 @@ static const char sbat[] _used_ _section_(".sbat") =
         SBAT_PROJECT ",1,The systemd Developers," SBAT_PROJECT "," PROJECT_VERSION "," PROJECT_URL "\n"
         SBAT_PROJECT "." SBAT_DISTRO "," STRINGIFY(SBAT_DISTRO_GENERATION) "," SBAT_DISTRO_SUMMARY "," SBAT_DISTRO_PKGNAME "," SBAT_DISTRO_VERSION "," SBAT_DISTRO_URL "\n";
 #endif
+
+EFI_STATUS secure_boot_enroll(EFI_FILE *root_dir, Config *config, ConfigEntry *entry) {
+        assert(root_dir);
+        assert(config);
+        assert(entry);
+
+        EFI_STATUS err = EFI_SUCCESS;
```

Comment:
(We normally don't initialize r/err).
```suggestion
        EFI_STATUS err;
```

Reply:
Not addressed.